HopsanCore/HopsanCore.pro

```diff
@@ -48,7 +48,7 @@ QMAKE_CXXFLAGS += -pedantic -Wno-long-long
 lessThan(QT_MAJOR_VERSION, 5){
   QMAKE_CXXFLAGS += -std=c++11
```

Comment:
Yes Qt4 does not support C++14. and its not supported by Hopsan either, I just keep this code here, in case. There is no need to deliberately remove Qt4 support, we just dont have to try and make it work anymore